apps/condo/domains/organization/schema/RegisterNewOrganizationService.js

```diff
@@ -2,6 +2,7 @@ const { createConfirmedEmployee, createOrganization, createDefaultRoles, pushOrg
 const { getById, GQLCustomSchema } = require('@condo/keystone/schema')
 const access = require('@condo/domains/organization/access/RegisterNewOrganizationService')
 const { createTrialSubscription } = require('@condo/domains/subscription/utils/serverSchema/ServiceSubscription')
+const { PropertyScope, createDefaultPropertyScopeForNewOrganization } = require('@condo/domains/scope/utils/serverSchema')
```

Comment:
can you mark it as tech dept? we need to create some tool in a future for that

Reply:
May be implement some Lodash concepts right here? Something like `get(organization, customScope, defaultScope)`, because I've got big concern over here.

We used same method in ticket statuses. And guess what? Nobody created a custom transition...
So why not store only custom objects here?

Reply:
No, this is not common scope for all organizations.
It should be different in each organization, because it can be explicitly changed and deleted in the interface